### PR TITLE
Add config flow to Random

### PR DESCRIPTION
**Pull request by joostlek**
## Proposed change

Add config flow to Random

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/random.markdown [hunk 1]
```diff
@@ -19,13 +20,23 @@ ha_integration_type: integration
 
 The `random` integration simply creates random values or state. This can be useful if you want to test automation rules or run an interactive demo. It generates a new state every time it is polled.
 
-## Binary Sensor
+The random binary sensor is creating random states (`true`, 1, `on` or `false`, 0, `off`).
+The random sensor is creating random sensor values (integers) out of a given range. Returned values form a [discrete uniform distribution](https://en.wikipedia.org/wiki/Discrete_uniform_distribution), meaning that each integer value in the range configured is equally likely to be drawn.
```
Comment (frenck) [inline source/_integrations/random.markdown]:
```suggestion
The random sensor creates random sensor values (integers) out of a given range. Returned values form a [discrete uniform distribution](https://en.wikipedia.org/wiki/Discrete_uniform_distribution), meaning that each integer value in the range configured is equally likely to be drawn.
```

File: source/_integrations/random.markdown [hunk 2]
```diff
@@ -19,13 +20,23 @@ ha_integration_type: integration
...
 
-The `random` binary sensor platform is creating random states (`true`, 1, `on` or `false`, 0, `off`).
+## Configuration
+The preferred way to configure timer helpers is via the user interface at **{% my helpers title="Settings > Devices & Services > Helpers" %}** and click the add button; next choose the {% my config_flow_start domain=random title="Random" %} option.
```
Comment (frenck) [inline source/_integrations/random.markdown]:
```suggestion
The preferred way to configure timer helpers is via the user interface at **{% my helpers title="Settings > Devices & Services > Helpers" %}** and select the add button; next, select the {% my config_flow_start domain=random title="Random" %} option.
```

File: source/_integrations/random.markdown [hunk 3]
```diff
@@ -19,13 +20,23 @@ ha_integration_type: integration
...
 
-### Configuration
+You can also click the following button to be redirected to the Helpers page of your Home Assistant instance.
```
Comment (frenck) [inline source/_integrations/random.markdown]:
```suggestion
You can also select the following button to be redirected to the Helpers page of your Home Assistant instance.
```

File: source/_integrations/random.markdown [hunk 4]
```diff
@@ -19,13 +20,23 @@ ha_integration_type: integration
...
 
-To enable the random binary sensor, add the following lines to your `configuration.yaml` file:
+{% my helpers badge %}
+
+To be able to add Helpers via the user interface you should have `default_config:` in your `configuration.yaml`, it should already be there by default unless you removed it. If you removed `default_config:` from your configuration, you must add `timer:` to your `configuration.yaml` first, then you can use the UI.
+
+## YAML configuration
+
+### Binary Sensor
```
Comment (frenck) [inline source/_integrations/random.markdown]:
```suggestion
### Binary sensor
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/random.markdown [hunk 5]
```diff
@@ -19,13 +20,21 @@ ha_integration_type: integration
 
 The `random` integration simply creates random values or state. This can be useful if you want to test automation rules or run an interactive demo. It generates a new state every time it is polled.
 
-## Binary Sensor
 
-The `random` binary sensor platform is creating random states (`true`, 1, `on` or `false`, 0, `off`).
+## Configuration
+The preferred way to configure timer helpers is via the user interface at **{% my helpers title="Settings > Devices & Services > Helpers" %}** and select the add button; next, select the {% my config_flow_start domain=random title="Random" %} option.
```
Comment (c0ffeeca7) [inline source/_integrations/random.markdown]:
```suggestion
The preferred way to configure random helpers is via the user interface at **{% my helpers title="Settings > Devices & Services > Helpers" %}** and select the add button; next, select the {% my config_flow_start domain=page.ha_domain title=page.title %} option.
```

File: source/_integrations/random.markdown [hunk 6]
```diff
@@ -19,13 +20,21 @@ ha_integration_type: integration
...
 
-### Configuration
+You can also select the following button to be redirected to the Helpers page of your Home Assistant instance.
```
Comment (c0ffeeca7) [inline source/_integrations/random.markdown]:
```suggestion
```

File: source/_integrations/random.markdown [hunk 7]
```diff
@@ -19,13 +20,21 @@ ha_integration_type: integration
...
 
-To enable the random binary sensor, add the following lines to your `configuration.yaml` file:
+{% my helpers badge %}
+
+To be able to add Helpers via the user interface you should have `default_config:` in your `configuration.yaml`, it should already be there by default unless you removed it. If you removed `default_config:` from your configuration, you must add `timer:` to your `configuration.yaml` first, then you can use the UI.
```
Comment (c0ffeeca7) [inline source/_integrations/random.markdown]:
```suggestion
To be able to add Helpers via the user interface you should have `default_config:` in your `configuration.yaml`, it should already be there by default unless you removed it. If you removed `default_config:` from your configuration, you must add `random:` to your `configuration.yaml` first, then you can use the UI.
```

File: source/_integrations/random.markdown [hunk 8]
```diff
@@ -19,13 +20,17 @@ ha_integration_type: integration
...
 The `random` integration simply creates random values or state. This can be useful if you want to test automation rules or run an interactive demo. It generates a new state every time it is polled.
 
-## Binary Sensor
 
-The `random` binary sensor platform is creating random states (`true`, 1, `on` or `false`, 0, `off`).
+## Configuration
+The preferred way to configure random helpers is via the user interface at **{% my helpers title="Settings > Devices & Services > Helpers" %}** and select the add button; next, select the {% my config_flow_start domain=page.ha_domain title=page.title %} option.
 
-### Configuration
+To be able to add Helpers via the user interface you should have `default_config:` in your `configuration.yaml`, it should already be there by default unless you removed it. If you removed `default_config:` from your configuration, you must add `random:` to your `configuration.yaml` first, then you can use the UI.
```
Comment (c0ffeeca7) [inline source/_integrations/random.markdown]:
```suggestion
To be able to add Helpers via the user interface, you should have `default_config:` in your `configuration.yaml`. It should already be there by default unless you removed it. If you removed `default_config:` from your configuration, you must add `random:` to your `configuration.yaml` first, then you can use the UI.
```

**Review by c0ffeeca7 — COMMENTED** [review]
missed one

**Review by c0ffeeca7 — DISMISSED** [review]
LGTM, @joostlek :+1:

**Review by frenck — APPROVED** [review]

**Pull request merged**